.github/workflows/run-kata-coco-tests.yaml

```diff
@@ -54,7 +46,7 @@ jobs:
       KUBERNETES: "vanilla"
       USING_NFD: "true"
       KBS: "true"
-      K8S_TEST_HOST_TYPE: ${{ matrix.k8s-test-host-type }}
+      K8S_TEST_HOST_TYPE: "baremetal"
```

Comment:
Hi @fidencio !

Do you still want to keep the baremetal-attestation and baremetal-no-attestation subset of tests in tests/integration/kubernetes/run_kubernetes_tests.sh ?

Otherwise, lgtm.

Reply:
I want to keep it there at least for a few more weeks.  Depending on who'll take over this CI maintainence I'll drop them from there as well.